ethereum/p2p/src/main/java/org/hyperledger/besu/ethereum/p2p/rlpx/connections/netty/DeFramer.java

```diff
@@ -185,8 +209,10 @@ protected void decode(final ChannelHandlerContext ctx, final ByteBuf in, final L
         ctx.close();
         connectFuture.completeExceptionally(
             new PeerDisconnectedException(disconnectMessage.getReason()));
+
       } else {
         // Unexpected message - disconnect
+
```

Comment:
these blank lines seem unnecessary - is this a style thing ?

Reply:
s.a.